sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpBodyType.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    /// <summary>
+    /// The types of AMQP bodies for an AMQP message.
+    /// </summary>
+    public enum AmqpBodyType
```

Comment:
Should this be `internal` ?

Reply:
We need to expose a way to get the body type out so the user would know which extension method to use.

Reply:
This is exposed via `GetAmqpBodyType()` extension method being added.